File: Getting-Practice-with-Python-and-Javascript/1-First-Introductory-Steps/Python/Excercises/program-7.py
# 7-Crea un programa que pida al usuario una cadena de caracteres y luego imprima el número de palabras en esa cadena que tienen más de 5 letras.


userInput = input('Ingrese una cadena de caracters: ')

lista = userInput.split(' ')
contador = 0
for c in lista:
    if len(c) > 5:
        contador += 1


print(contador)

File: Getting-Practice-with-Python-and-Javascript/1-First-Introductory-Steps/Javascript/Excercises/program-7.js
// 7-Crea un programa que pida al usuario una cadena de caracteres y luego imprima el número de palabras en esa cadena que tienen más de 5 letras.


let userInput = prompt('Ingresa una cadena de palabras: ')
let contador = 0;

const lista = userInput.split(' ')

for (let i = 0; i < lista.length; i++) {

    // console.log(lista[i].length)
    if (lista[i].length > 5) {
        contador += 1;
    }
}

console.log(contador)
